utils/node_local_test.py

```diff
@@ -4068,6 +4068,9 @@ def test_pil4dfs(self):
         with open(file1, 'w') as fd:
             fd.write('Hello World!')
 
+        # hexdump to check file
```

Comment:
> wrong-spelling-in-comment, Wrong spelling of a word 'hexdump' in a comment:

You could either change the comment or update `words.dict`

Reply:
Thank you! I will update words.dict later once all CI tests pass.